wee_slack.py

```diff
@@ -369,7 +369,7 @@ def buffer_input_cb(b, buffer, data):
     channel = channels.find(buffer)
     if not channel:
         return w.WEECHAT_RC_OK_EAT
-    reaction = re.match("(\d*)(\+|-):(.*):", data)
+    reaction = re.match("^\s*(\d*)(\+|-):(.*):\s*$", data)
```

Comment:
I don't think we should eat leading whitespace, since it's a way for people to escape when explaining something.